locale/en.json

```diff
@@ -39,6 +39,7 @@
     "copy": "Copy"
   },
   "sidebar": {
+    "nodeFilter": "Node filter",
```

Comment:
No idea. I still think, we never use the term name, hostname and it is not a search. All terms about node/nodes and what i can do with this input is filter them. -> For me a Node filter